5thColumn/kibana5-html-plugin
# A few css changes to make the header bigger

### Testing Instructions
1. Pull this branch down into your `plugins` folder.
1. Pull down this [branding plugin branch](https://github.com/5thColumn/kibana5-plugin-5c-branding/tree/make-headers-bigger) specifically the `make-headers-bigger` branch into your `plugins` folder.
1. Add an HTML visualization to a dashboard in your local Kibana, and add the following code
```
<script data-name="myscript">
var thisScript = document.querySelector('script[data-name="myscript"]');
fiveC.header(
'title', 
'https://5thcolulmn.net/cdn/hoops.jpeg',
thisScript);
</script>
```

If everything works correctly, you should see the following.  The image should be nearly all the way across the screen, and around `176px` tall :)

![screen shot 2017-03-03 at 4 13 22 pm](https://cloud.githubusercontent.com/assets/3399841/23570723/972ae190-0033-11e7-9065-5a61dbc0a2fd.png)

This PR address [this](https://github.com/5thColumn/kibana5-plugin-5c-branding/issues/21) issue and [this](https://github.com/5thColumn/kibana5-plugin-5c-branding/issues/20) one
